drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/widgets/ScenarioGridTest.java

```diff
@@ -54,6 +54,7 @@
 import org.uberfire.ext.wires.core.grids.client.widget.grid.renderers.grids.impl.BaseGridRendererHelper;
 import org.uberfire.ext.wires.core.grids.client.widget.grid.selections.SelectionExtension;
 
+import static org.drools.workbench.screens.scenariosimulation.client.TestProperties.GRID_COLUMN_GROUP;
```

Comment:
@yesamer 
Even if not part of your modifications,  since you touched this class , please move all the java static values (String, Integer, int, List, etc) to the TestProperties class.

Reply:
Done!

Reply:
@yesamer 
Thx!